package.json

```diff
@@ -1,22 +1,20 @@
 {
```

Comment:
I moved `http-server` into devDependencies, since I don't think we need it for anything in the deployment, and so then I got rid of the fully-qualified executable paths for our scripts since NPM will inject those into the PATH when it runs the scripts.